src/Utils/getFilterVideos.jsx

```diff
@@ -0,0 +1,19 @@
+export const getFilterVideos = (state) => {
+  const { categoryFilter, originalVideos, searchValue } = state;
+  let filterVideos = originalVideos;
+  if (categoryFilter !== "All Videos") {
+    filterVideos = filterVideos.filter(
+      (video) => video.category === categoryFilter
+    );
+  }
```

Comment:
i think you can write one line of code here 
  filterVideos = categoryFilter !== ? filterVideos.filter(
      (video) => video.category === categoryFilter
    ) : filterVideos;
    you should try this

Reply:
here no else part, so not needed